code/drasil-lang/lib/Language/Drasil/Chunk/NamedIdea.hs

```diff
@@ -37,65 +37,66 @@ class NamedIdea c => Idea c where
 --
 -- Ex. Anything worth naming must start out somewhere. Before we can assign equations
 -- and values and symbols to something like the arm of a pendulum, we must first give it a name. 
-data NamedChunk = NC {
-  _uu :: UID,
-  _np :: NP
-}
-makeLenses ''NamedChunk
+-- data NamedChunk = NC {
+--   _uu :: UID,
+--   _np :: NP
+-- }
+-- makeLenses ''NamedChunk
 
--- | Equal if 'UID's are equal.
-instance Eq        NamedChunk where c1 == c2 = (c1 ^. uid) == (c2 ^. uid)
--- | Finds the 'UID' of the 'NamedChunk'.
-instance HasUID    NamedChunk where uid = uu
--- | Finds the term ('NP') of the 'NamedChunk'.
-instance NamedIdea NamedChunk where term = np
--- | Finds the idea of a 'NamedChunk' (always 'Nothing').
-instance Idea      NamedChunk where getA _ = Nothing
+-- -- | Equal if 'UID's are equal.
+-- instance Eq        NamedChunk where c1 == c2 = (c1 ^. uid) == (c2 ^. uid)
+-- -- | Finds the 'UID' of the 'NamedChunk'.
+-- instance HasUID    NamedChunk where uid = uu
+-- -- | Finds the term ('NP') of the 'NamedChunk'.
+-- instance NamedIdea NamedChunk where term = np
+-- -- | Finds the idea of a 'NamedChunk' (always 'Nothing').
+-- instance Idea      NamedChunk where getA _ = Nothing
 
 -- TODO: Add in function to check UIDs (see #2788).
 -- TODO: Any constructor that takes in a UID should be built off of this one so that
 -- the UID may be checked by the first TODO.
 -- | 'NamedChunk' constructor, takes a 'String' for its 'UID' and a term.
-nc :: String -> NP -> NamedChunk
-nc s = NC (mkUid s)
+nc :: String -> NP -> Maybe String -> IdeaDict
```

Comment:
This change is incorrect: the signature of `nc` should not have changed. Same with `ncUID`.